turingschool-examples/ideabox-boilerplate
# Feature/display local storage

### What does this PR do?
- Adds functionality to the local storage so when a card is deleted and page is refreshed the other saved cards are still available in storage. 
### What does it fix?
- NA
### Is this a feature or a fix?
- Feature
### Where should the reviewer start?
- Main.js lines 13 / 17-25 / 
### How should this be tested?
- Open file in browser. Type text in fields and save multiple times. Open inspector and navigate to Application/ Local Storage. Now delete any previously made cards and watch the local storage remove the deleted card. Refresh page and check to see if local storage is still showing saved cards with Key.
